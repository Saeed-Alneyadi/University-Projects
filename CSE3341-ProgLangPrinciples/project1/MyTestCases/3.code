

\n
procedure FACTORIAL begin
    int x := 1
    int y := 1;
    
    intint z:= x + y;

    while (true) (
        x :=y;
        y = z;
        z = x +y;
    end

;