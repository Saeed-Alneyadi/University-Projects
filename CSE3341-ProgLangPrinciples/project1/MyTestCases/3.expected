ERROR: Unknown Script "\"
ERROR: Unknown Script "\"
